### DISCUSSION TITLE
Welcome to gvsbuild Discussions!

### DISCUSSION THREAD

**Discussion by danyeaw** [wingtk/gvsbuild#947] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by g40** [comment]
So, first off thanks. This is a pretty monumental piece of work, executed really well.

Which leads on to the second point: is there a minimal demo application which shows proof of life? Ideally something that generated a VS project file to show debugging etc. works as expected? 

TIA.
